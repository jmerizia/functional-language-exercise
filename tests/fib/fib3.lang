(def fib n (do
    (let a (fib (- n 1)))
    (let b (fib (- n 2)))
    (match n
        0   : 0
        1   : 1
        ANY : (+ a b)
    )
))
(print (fib 82))
(print (fib 83))
(print (fib 84))
