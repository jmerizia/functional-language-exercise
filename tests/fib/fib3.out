61305790721611591
99194853094755497
160500643816367088

